frontend_lib/src/component/PopinFixed/PopinFixedHeader.jsx

```diff
@@ -109,6 +118,7 @@ class PopinFixedHeader extends React.Component {
                 <div
                   className={classnames('wsContentGeneric__header__title', `${customClass}__header__title`)}
                   title={rawTitle}
+                  id='rawTitle'
```

Comment:
```suggestion
                  id={rawTitle}
```

Reply:
Like that if we have two titles in the same page, we don't have the same id.

Reply:
Having the same id actually prevents having two popovers at the same time, because the last clicked rawTitle replaces the previous one, instead of showing two (I tried it out)

Reply:
`id` is an unique identifier, so two HTML element should not have the same `id`. Or else the code is not following DOM standard (https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) and it's (theoretically) not a valid HTML.

Reply:
`id` is an unique identifier, so two HTML element should not have the same `id`. Or else the code is not following DOM standard (https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) and it's (theoretically) not a valid HTML.

Reply:
After speaking with @Skylsmoi 
The choice is to not merge this feature (add pop hover). I'd suggest to create another branch from commit `c0cacba` (I mean after this one, just before `b1573fc`) so we can safely merge previous modification without losing all our work and keep/use the second branch for R&D about a clean solution.

Since the `id` will be created from the title of a content/space, we'll require to have a slug function to remove bad characters, etc.